### PR TITLE
add docs for launching mainnet

### PR DESCRIPTION
**Pull request by KeefeL**
## Description

Add docs for launching Greenfield Mainnet, and change the outdated contents.

## Rationale

Make the docs correct.

### CODE REVIEW

File: docs/api/endpoints.md [hunk 1]
```diff
@@ -4,33 +4,63 @@ icon: http
...
+- **Blockchain RPC:**
+  - `https://greenfield-chain-ap.bnbchain.org`
+  - `https://greenfield-chain-us.bnbchain.org`
+  - `https://greenfield-chain-eu.bnbchain.org`
+  - `https://greenfield-chain.bnbchain.org`
 
+- **Open API:**
+  - `https://greenfield.bnbchain.org/openapi`
+
+- **SP Endpoints:**
```
Comment (zjubfd) [inline docs/api/endpoints.md:19]: can we hightlight that this is only the genesis storage provide, for the full list, user can go to the greenfield scan to check

File: docs/guide/core-concept/gas-fees.md [hunk 2]
```diff
@@ -174,6 +175,7 @@ documentation.
 | cosmos.bank.v1beta1.MsgMultiSend           | 800 + 800 per item | 5 gwei    | $0.0012 per item |
 | cosmos.feegrant.v1beta1.MsgGrantAllowance  | 800 + 800 per item | 5 gwei    | $0.0012 per item |
```
Comment (zjubfd) [inline docs/guide/core-concept/gas-fees.md:176]: Can you double check the fee table here, I think we change some of the price.

File: docs/release-notes/features.md [hunk 3]
```diff
@@ -11,24 +11,25 @@ the key features and their development status on Greenfield. Please stay tuned.
...
\ No newline at end of file
+|  Expiration time for group member  | Greenfield supports setting expired time for group members                                                                                                                                                                                                                                                                                                                                                                                 | Mainnet |
+ |          Link with opBNB           | Build native cross-chain bridge between Greenfield and opBNB                                                                                                                                                                                                                                                                                                                                                                               |   Dev   |
```
Comment (zjubfd) [inline docs/release-notes/features.md:35]: `Greenfield as DA layer` we can add this, in design status.

File: docs/guide/core-concept/gas-fees.md (hunk 2 above)
Comment (KeefeL) [reply docs/guide/core-concept/gas-fees.md:176]: This is the newest one, we have changed it before.

File: docs/release-notes/features.md (hunk 3 above)
Comment (KeefeL) [reply docs/release-notes/features.md]: fixed

File: docs/api/endpoints.md (hunk 1 above)
Comment (KeefeL) [reply docs/api/endpoints.md:19]: sure, fixed

**Review by zjubfd — APPROVED** [review]

**Pull request merged**